### DISCUSSION TITLE
Perovskites

### DISCUSSION THREAD

**Discussion by JJ-882019** [pycalphad/pycalphad#338] [Q&A] [answered]
Thanks for the resource!  I'm looking at modeling the reduction behavior in perovskite structures ABX3, with the focus on vacancy formation. Is this possible using pycalphad?

Also, not sure if the TDB file Ive been using is appropriate (I'm assuming not)? 

Thanks again.

**Comment by richardotis** [comment, marked as answer]
This type of investigation is definitely possible using pycalphad. The key is you need a database for the A-B-X perovskite system of interest. The TDBDB is a good place to start: https://avdwgroup.engin.brown.edu/
It's not comprehensive though, and you may need to perform a more detailed literature search. A more advanced approach would be to build your own database for the investigation, which is possible using ESPEI: https://espei.org/en/latest/
